### DISCUSSION TITLE
识别表格输出的html结果有问题

### DISCUSSION THREAD

**Discussion by zaoa0** [RapidAI/TableStructureRec#55] [Q&A] [answered]
这是原图：
![屏幕截图 2024-10-23 174901](https://github.com/user-attachments/assets/4e2728fd-c666-498b-80dc-c59cd7ef40a1)
识别结果：
![image](https://github.com/user-attachments/assets/8b9d1149-44fd-42bf-a04b-028079b2fdb8)
是表格识别模型输出结果转换成html过程中出的问题吗，看有线模型的检测结果是正常的

**Comment by Joker1212** [comment]
> 这是原图： ![屏幕截图 2024-10-23 174901](https://private-user-images.githubusercontent.com/116989198/380565097-4e2728fd-c666-498b-80dc-c59cd7ef40a1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAwOTcwMDcsIm5iZiI6MTczMDA5NjcwNywicGF0aCI6Ii8xMTY5ODkxOTgvMzgwNTY1MDk3LTRlMjcyOGZkLWM2NjYtNDk4Yi04MGRjLWM1OWNkN2VmNDBhMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDI4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAyOFQwNjI1MDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01OTEwNGYzMTc5Yjc0NzQ2NTZlNGE0NmZmMWZkYjRhODllNWY5NWJlM2E4ZTg5MWY5OWFlZTE0YWYyNzUxNTkyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.TluJSWkJC4sVR1oUIfGIRwd9HIMrXWlkhfNoJUeCEag) 识别结果： ![image](https://private-user-images.githubusercontent.com/116989198/380564514-8b9d1149-44fd-42bf-a04b-028079b2fdb8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAwOTcwMDcsIm5iZiI6MTczMDA5NjcwNywicGF0aCI6Ii8xMTY5ODkxOTgvMzgwNTY0NTE0LThiOWQxMTQ5LTQ0ZmQtNDJiZi1hMDRiLTAyODA3OWIyZmRiOC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDI4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAyOFQwNjI1MDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01YWY2NDU3N2ZkNjc0ZjRiNGUzMGQyYjViMzZiMzlhODY2YjQyNmNlMTcxZGM5NmM2YjViZjg2NzVlODIzZTU4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.2hJozGavfIZb_YJtx36FRuQc6mT1hEYNuZ_-goDFuGY) 是表格识别模型输出结果转换成html过程中出的问题吗，看有线模型的检测结果是正常的


感谢反馈哈，确实有bug，非常感谢，帮助我找到问题，能再提升识别精度！

**Comment by Joker1212** [comment, marked as answer]
已经解决了哈，可以拉取最新的包试试
